wiz-broker/templates/wiz-broker-deployment.yaml

```diff
@@ -86,9 +86,15 @@ spec:
           {{- with .Values.global.broker.customVolumeMounts }}
             {{- toYaml . | nindent 10 }}
           {{- end }}
-          args: [
-            {{ $connectorDataFilePath }}
-          ]
+          {{- if .Values.broker.debug }}
```

Comment:
1. I would like to avoid spawning bash and having branches on the entrypoint, nonetheless the dependency on `/bin/sh`
2. We control this via the environment variable `LOG_LEVEL=debug`
3. If you'd like, we can also `set -x` as part of the entrypoint and not spawning bash.

Reply:
Great points, fixed!